consoleme/celery_tasks/celery_tasks.py

```diff
@@ -573,8 +573,22 @@ def cache_policies_table_details() -> bool:
     if all_s3_errors:
         s3_errors = json.loads(all_s3_errors)
 
+    allowed_tags = config.get("roles.allowed_tags")
+    allowed_arns = config.get("roles.allowed_arns")
     for arn, role_details_j in all_iam_roles.items():
         role_details = ujson.loads(role_details_j)
+
+        allowed = True
+
+        if allowed_tags and {"consoleme":"managed"} not in role_details.get("tags"):
```

Comment:
@jaydhulia This part I'm not comfortable with since i'm not referencing a struct with defined attributes. I'm not 100% sure how to grab the tags from the `role_details` in this part

Reply:
What about making `allowed_tags` into a list of allowed tags (list of key<>value pair in YAML), and seeing that any of the allowed tags are present in the role's tags?
I am not sure if it would make sense to match ALL allowed tags in order to count, or just one of them. To me, it makes sense to include roles that have atleast 1 allowed tag from the config. 
Thoughts @rms1000watt ?